docs/self-managed/identity/user-guide/making-identity-production-ready.md

```diff
@@ -12,6 +12,11 @@ As Keycloak is an external-based dependency of the Identity application, we reco
 [Keycloak's documentation on production configuration](https://www.keycloak.org/server/configuration-production) to
 make sure your Keycloak instance is production-ready.
 
+### Backup
+
+To ensure recovery is possible we recommend regularly backing up the database that supports Keycloak.
+For more information about backups, please consult the documentation of your chosen provider.
```

Comment:
One thing that I hadn't spotted before, and it only really applies to our Helm charts is that there is a relevant section in the `:latest` docs for scheduling backups via the Keycloak Operator https://www.keycloak.org/docs/latest/server_installation/#_backups-local-cr - I think this would be good to reference here. 

Regarding the general backup advice Keycloak only make reference to backups as part of an upgrade procedure which might be a little confusing given our context. 

Given that my preference would be to break this section down a little more and have a section for "how to backup when deploying with our helm charts <link>" and then another general section that suggests if you configure Keycloak via another method i.e. docker-compose locally then reference the provider for advice. 

wdyt?

Reply:
The [upper section ](https://www.keycloak.org/docs/latest/server_installation/#_backup-cr) states that the method is deprecated, so I would not link to it. I can not find any batter example in the docs, tho.

> Backup CR is deprecated and could be removed in future releases.

I like the idea with two sections. When using our charts we could link to Postges backup docs: https://www.postgresql.org/docs/current/backup.html. When using an own Keycloak or a different database we could just link the Keycloak docs and keep the generic provider advice (the one you have already added)

Reply:
I have now addressed your comments :)

At this time I have decided against linking the Keycloak documentation which mentions backing up because I believe the context is confusing and its an item in a list of instructions that is not easy to anchor to.